### DISCUSSION TITLE
URL like `shared` that plays directly, for automation

### DISCUSSION THREAD

**Discussion by bilogic** [navidrome/navidrome#3229] [Ideas]
1. A shared URL plays directly
2. But I can't get the same effect for a radio
3. Navidrome will load, but nothing plays until I click around the right buttons

Would be great because I would like to automate the radio playing at a certain time daily.

Thank you!

**Comment by deluan** [comment]
You mean, the player that appears when you open the share URL could potentially auto-play? Maybe an auto-play option in the share itself?

**Reply by bilogic** [reply]
1. The URL of a `shared song` auto plays
2. Is there a URL for a `radio` to also auto play?

Why? I want to automate the radio playing at fixed times daily. It would get Chrome to load the (yet to be implemented) navidrome URL and that radio station should start playihng.

**Reply by deluan** [reply]
Ah, I see. Why not use the actual Stream URL for that radio?

**Reply by bilogic** [reply]
Oh, single source of music control.

An actual stream will open up in a tab/window of its own and if I needed to play something else in Navidrome, I need to find and close this separate tab, open Navidrome instead of just going to Navidrome, which I have installed as an app.
